.github/workflows/ci.yml

```diff
@@ -30,15 +30,11 @@ jobs:
             arch: x64
             max-cxx-std: 17
             args: "-DPYBIND11_FINDPYTHON=ON"
-          - runs-on: macos-latest
-            python: 3.7
-            arch: x64
-            max-cxx-std: 17
-            args: "-DPYBIND11_FINDPYTHON=ON"
           - runs-on: windows-2016
             python: 3.7
             arch: x86
-            max-cxx-std: 14
+            max-cxx-std: 17
+            args: "-DCMAKE_CXX_FLAGS='//permissive- //DWIN32 //D_WINDOWS //W3 //GR //EHsc'"
```

Comment:
Don't seem to be able to inject a single flag (or flags properly) to MSVC's build...

Reply:
Must be something with GHA, though. I just tried, and I can perfectly run `cmake .. -DPYTHON_EXECUTABLE=C:\Python38\python.exe -DCMAKE_CXX_FLAGS=/permissive-` :-/

Something with thing being escaped?

Reply:
It is definitely messing with escaping.